.devcontainer/Dockerfile.default

```diff
@@ -0,0 +1,19 @@
+FROM ghcr.io/insight-platform/savant-rs-py313:latest AS dev
```

Comment:
Use `savant-latest` not `latest`. It guarantees that there is a Savant build supporting this library version.